test/integration/roles/test_amazon_rds/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+# create and kill an RDS
+
+# Improvements expected in later versions of this test case
+
+# use present instead of create
+# use absent instead of delete
+# use SSD
+
+
+# Start with creation - N.B. if the RDS already exists from a previous run this
+# may not change anything so we don't insist on that
+
+  - name: create RDS
+    tags: test_amazon_rds
+    local_action:
+      module: rds
+      region: "{{ ec2_region }}"
+      command: create
+      instance_name: "{{ resource_prefix }}-rds"
+      db_engine: postgres
+      size: 10
+      instance_type: "db.t2.micro"
+      username: "{{ rds_master_db_user }}"
+      password: "{{ rds_db_master_password }}"
+      publicly_accessible: no
+      wait: yes
+      wait_timeout: 1200 # 600 doesn't cut it!!
+    environment:
+      EC2_REGION: '{{ec2_region}}'
```

Comment:
Apologizing to hijack this train of thought, but @willthames, but I'm curious why you think we should use profile for integration tests as I'm working on the issue linked above and will be working on more after that. Our integration test credentials.yml doesn't currently have an ec2_profile var. I agree though about not using environment variables if we aren't testing them specifically. Since we're repeating the credentials for ever single task, I wish there was a way to extrapolate them. We can't just use export AWS_ACCESS_KEY because the integration tests are using the [credentials.yml file](https://github.com/ansible/ansible/blob/devel/test/integration/README.md#cloud-tests). I guess we could use the shell module and create a testing profile?